en/middleware/micrortps.md

```diff
@@ -89,6 +92,8 @@ By default the *Client* runs for 10000 loops and then stops. To run the *Client*
 micrortps_client start -l -1
 ```
 
+> **Note** The PX4 Firmware initialisation code may in future automatically start the *Client* as a permanent daemon process. In the meantime you will need to start the client manually.<!-- at that point, most of this section would move into the "manual generation" doc: https://github.com/PX4/Firmware/pull/7663#issuecomment-317928506 -->
```

Comment:
@VicenteEprosima FYI, you stated that there is no plan to make starting the client automatically. This modification changes the statement that "in future we will do this" to "this is something we may do"